PartPreviewWindow/View3D/MeshViewerWidget.cs

```diff
@@ -510,6 +510,7 @@ private void DrawObject(IObject3D object3D, List<Object3DView> transparentMeshes
 						{
 							selectionColor = accentColor.Blend(Color.White, Quadratic.InOut((secondsSinceSelectionChanged - .25) * 4));
 						}
+						Invalidate();
```

Comment:
@larsbrubaker  - This call was removed in the prior commit, due to MatterHackers/MCCentral#3347. Was it restored on purpose or by accident?